jerry-core/ecma/operations/ecma-conversion.c

```diff
@@ -664,193 +663,183 @@ ecma_op_to_property_descriptor (ecma_value_t obj_value, /**< object value */
                                                                                   if return value is normal
                                                                                   empty completion value */
 {
-  ecma_value_t ret_value = ECMA_VALUE_EMPTY;
-
   /* 1. */
   if (!ecma_is_value_object (obj_value))
   {
-    ret_value = ecma_raise_type_error (ECMA_ERR_MSG ("Expected an object."));
+    return ecma_raise_type_error (ECMA_ERR_MSG ("Expected an object."));
   }
-  else
-  {
-    ecma_object_t *obj_p = ecma_get_object_from_value (obj_value);
 
-    /* 2. */
-    ecma_property_descriptor_t prop_desc = ecma_make_empty_property_descriptor ();
+  ecma_object_t *obj_p = ecma_get_object_from_value (obj_value);
 
-    /* 3. */
-    ECMA_TRY_CATCH (enumerable_prop_value,
-                    ecma_op_object_find (obj_p, ecma_get_magic_string (LIT_MAGIC_STRING_ENUMERABLE)),
-                    ret_value);
+  /* 2. */
+  ecma_property_descriptor_t prop_desc = ecma_make_empty_property_descriptor ();
```

Comment:
This can be move before the `if (ecma_is_value_found (enumerable_prop_value))` as we are only using the variable from there.